source/jsonnet/northern-ireland/individual/blocks/identity-and-health/religion.jsonnet

```diff
@@ -40,12 +40,7 @@ local question(title) = {
         {
           label: 'Other',
           value: 'Other',
-          detail_answer: {
-            id: 'religion-answer-other',
-            type: 'TextField',
-            mandatory: false,
-            label: 'Please specify religion, religious denomination or body',
-          },
+          description: otherReligionDescription,
```

Comment:
Proxy variant has an `Any other religion` label in the spreadsheet, might be worth checking.

Reply:
Thats a mistake in the spreadsheet, I checked with Donna yesterday

Reply:
Thanks👍